content/project/trademark/approved-usage.adoc

```diff
@@ -185,4 +186,14 @@ link:http://meetings.jenkins-ci.org/jenkins-meeting/2019/jenkins-meeting.2019-11
 and
 link:http://meetings.jenkins-ci.org/jenkins-meeting/2019/jenkins-meeting.2019-11-20-18.00.log.html[raw]
 
+|Oleg Nenashev |Jenkins RU
+|
+link:https://groups.google.com/g/jenkinsci-dev/c/yq_FrEf9d28[Mailing list] +
+link:https://docs.google.com/document/d/11Nr8QpqYgBiZjORplL_3Zkwys2qK1vEvK-NYyYa4rzg/edit#heading=h.u3hysigdv557[Meeting minutes]
+
+|link:https://www.techmatrix.co.jp/en/[TechMatrix] |TechMatrix Jenkins Support
```

Comment:
This is wrong, we approved the other usage.

Reply:
Right, will fix